source/api_documentation/network_integration/affiliates/_put_affiliate.rst

```diff
@@ -6,7 +6,7 @@
 
   Endpoint:
 
```

Comment:
In case anyone wonders, this is unrelated to the story but was a typo Bobby found so I threw it in.